dub.json

```diff
@@ -10,8 +10,35 @@
 
     "targetType": "executable",
     "targetPath": "build",
+    "excludedSourceFiles": [ "source/scpp/*.d" ],
+    "preBuildCommands": [ "source/scpp/build.d" ],
+    "sourceFiles": [
```

Comment:
This should be automated, and it's ridiculous that `dub` does not allow globbing here.
I used `ls -1 build/ | xargs -I '{}' echo '"source/scpp/build/{}"',` to generate this list.